src/fragments/lib/push-notifications/react-native/request_permissions/request-permissions.mdx

```diff
@@ -0,0 +1,83 @@
+- **GRANTED** - Permissions have been granted by the user. No further actions are needed and their app is ready to display notifications.
+- **DENIED** - Permissions have been denied by the user. Further attempts to request permissions will no longer trigger a permission dialog. Your app should now either degrade gracefully or prompt your user to grant the permissions needed in their device settings.
+
+<Callout>
+  If you use TypeScript for your development, the string values returned can be
+  represented as PushNotificationPermissionStatus enum members as well.
+</Callout>
+
+```js
+const status = await Notifications.Push.getPermissionStatus();
+// 'SHOULD_REQUEST' | 'SHOULD_EXPLAIN_THEN_REQUEST' | 'GRANTED' | 'DENIED'
+```
+
+## Request permissions
+
+Once you have determined the current status of permissions requires you to request permissions from you user, you can call `requestPermissions()` to make that request.
+
+Amplify requests all supported notification permissions by default. But you can also choose not to request specific permissions.
+
+<Callout>
+  It is recommended that you specify these permissions if needed but it is
+  important to note that they are ignored by Android
+</Callout>
+
+- **Alert**: When set to true, requests the ability to display notifications to the user.
+- **Sound**: When set to true, requests the ability to play a sound in response to notifications.
+- **Badge**: When set to true, requests the ability to update the app's badge.
+
+```js
+const permissions = {
+  // permissions are true by default
+  // alert: true
+  sound: false,
+  badge: false
+};
+
+const result = await Notifications.Push.requestPermissions(permissions);
+// true if granted (or already granted), false otherwise
+```
+
+## Sample permissions flow
+
+Use `getPermissionStatus()` and `requestPermissions()` together handle permission request flows. Below is a sample implementation of the expected logic.
+
+<Callout>
+  Remember, if you use TypeScript for your development, you can use the
+  PushNotificationPermissionStatus enum for the comparisons as well!
+</Callout>
+
+```js
+function handlePermissions() {
+  const status = await Amplify.Notifications.Push.getPermissionStatus();
+  if (status === 'GRANTED') {
+    // no further action is required, user has already granted permissions
+    return;
+  }
+  if (status === 'DENIED') {
+    // further attempts to request permissions will no longer do anything
+    myFunctionToGracefullyDegradeMyApp();
+    return;
+  }
+  if (status === 'SHOULD_REQUEST') {
+    // go ahead and request permissions from the user
+    await Amplify.Notifications.Push.requestPermissions();
+  }
+  if (status === 'SHOULD_EXPLAIN_THEN_REQUEST') {
+    // you should display some explanation to your user before requesting permissions
+    await myFunctionExplainingPermissionsRequest();
+    // then request permissions
+    await Amplify.Notifications.Push.requestPermissions();
+  }
+}
+```
```

Comment:
Another code block where we use `Amplify.Notifications...` instead of just `Notifications...`

Reply:
Ahh good catch. Will update